packages/spruce-next-helpers/src/skillskit/next/PageWrapper.js

```diff
@@ -314,12 +342,14 @@ const PageWrapper = Wrapped => {
 			if (this.props.config.DEV_MODE) {
 				return (
 					<Container>
+						<FontLoader fonts={[...fonts]} />
```

Comment:
`fonts={fonts}` (unless you have another reason to spread it)

Reply:
updated